cmd/github-ldap-user-group-creator/main.go

```diff
@@ -6,14 +6,14 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-	"strings"
 
 	"github.com/sirupsen/logrus"
 
 	"k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
I am surprised that this PR also changes the `github-ldap-user-group-creator` and I don't know why. Could you accompany the PR with the description about what is the new tool supposed to do, and why is the PR accompanied with these seemingly unrelated changes?

Reply:
These questions have been addressed in https://docs.google.com/document/d/1Q003WYYHpzKj5y232QLW-BgA5pTrDPXRjWShCesWlc8/edit#

I will update the readme after our arch call.

Reply:
The fact that these questions were addressed somewhere else is not entirely useful. I find it reasonable to require large PRs like this to carry one or two paragraph description of the change. For you it may seem like repeating the content from elsewhere, but PR descriptions is the thing that the reviewers will read, not the google document.

Reply:
I mixed up two things in the comments: PR description and ReadMe for the tools.
I missed the former (my mistake) and wanted to added the latter after arch meeting (in case something is changed there).

Let me add both now to make the review easier.

Reply:
Done.